APCAppCore/APCAppCore/UI/Onboarding/APCShareViewController.m

```diff
@@ -196,8 +196,20 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                 MFMessageComposeViewController *messageComposer = [[MFMessageComposeViewController alloc] init];
                 messageComposer.messageComposeDelegate = self;
                 messageComposer.body = self.shareMessage;
+
+                if (_textMessageNavTintColor)
```

Comment:
Nit: could you explicitly check against nil? I think it tends to read a bit faster when maintaining code later on. Thanks!